plugin/build.gradle.kts

```diff
@@ -11,6 +11,7 @@ plugins {
     kotlin("jvm") version "1.3.61"
     id("com.gradle.plugin-publish") version "0.10.1"
     id("io.gitlab.arturbosch.detekt") version "1.4.0"
+    id("maven-publish")
```

Comment:
This is unfortunately needed to deploy to a local repository. More below.

Reply:
There is actually already `maven-publish` applied just a couple of lines above :)

Reply:
Ooops, sorry, will remove